gatling-jms/src/main/scala/io/gatling/jms/action/JmsRequestSend.scala

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2011-2016 GatlingCorp (http://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gatling.jms.action
+
+import javax.jms.Message
+
+import io.gatling.commons.stats.{ KO, OK, Status }
+import io.gatling.commons.util.TimeHelper.nowMillis
+import io.gatling.commons.validation.{ Failure, SuccessWrapper, Validation }
+import io.gatling.core.action.{ Action, ExitableActorDelegatingAction, ValidatedActionActor }
+import io.gatling.core.check.Check
+import io.gatling.core.session.{ Expression, Session, resolveOptionalExpression }
+import io.gatling.core.stats.StatsEngine
+import io.gatling.core.stats.message.ResponseTimings
+import io.gatling.core.util.NameGen
+import io.gatling.jms.JmsCheck
+import io.gatling.jms.client.JmsClient
+import io.gatling.jms.protocol.JmsProtocol
+import io.gatling.jms.request.{ BytesJmsMessage, JmsAttributes, MapJmsMessage, ObjectJmsMessage, TextJmsMessage }
+
+import akka.actor.ActorSystem
+import akka.actor.Props
+/**
+ * Core JMS Action to handle Send
+ *
+ * This handles the core "send"ing of messages. Gatling calls the execute method to trigger a send.
+ */
+object JmsRequestSend extends NameGen {
+
+  def apply(attributes: JmsAttributes, protocol: JmsProtocol, system: ActorSystem, statsEngine: StatsEngine, next: Action) = {
+    val actor = system.actorOf(JmsRequestSendActor.props(attributes, protocol, statsEngine, next))
+    new ExitableActorDelegatingAction(genName("jmsRequest"), statsEngine, next, actor)
```

Comment:
I don't think we need an actor here as it doesn't have mutable internal state.
A simple `ExitableAction` probably suffice.